src/main/java/org/openrewrite/javascript/JavaScriptParser.java

```diff
@@ -15,10 +15,37 @@
  */
 package org.openrewrite.javascript;
 
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.UncheckedIOException;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.LinkOption;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
```

Comment:
It looks like you have non-standard settings for the imports. The import order is different and there are no star-imports when there are at least 5 imports from a package. Please reset your Java formatter settings in IDEA to the defaults.